# Análisis de Usabilidad SUS - SGROAS
**Bloque C.3** | Fuente: `docs/mediciones/sus/sus-raw.csv` (n=15, P01..P15) | Método: Brooke 1996, IC95 con t de Student, escala adjetiva Bangor et al. 2009
Al abrir este cuaderno se ve el último run que produjo los números del documento (D.1). Rutas relativas a la raíz del repo. El CSV se lee como `latin-1` porque mezcla `Sí`/`Si` en consentimiento.

In [1]:
import csv, pathlib
raw = pathlib.Path("docs/mediciones/sus/sus-raw.csv").read_text(encoding="latin-1").splitlines()
rows = list(csv.DictReader(raw))
print(f"Participantes: {len(rows)}")
print("Codigos: " + ", ".join(r["codigo"] for r in rows))
scores = [float(r["sus_score"]) for r in rows]
print(f"Media verificada sus_score: {sum(scores) / len(scores):.1f}")


Participantes: 15
Codigos: P06, P07, P08, P09, P10, P01, P02, P03, P04, P05, P11, P12, P13, P14, P15
Media verificada sus_score: 68.5


In [2]:
import json, pathlib
est = json.loads(pathlib.Path("docs/mediciones/sus/estadisticas-sus.json").read_text(encoding="utf-8"))
print(f"n={est['n']} media={est['media']:.1f} DT={est['desviacion_tipica']:.3f} SE={est['error_estandar']:.4f} t(gl=14)={est['t_critico']:.4f}")
print(f"IC95=[{est['ic95_inf']:.2f}; {est['ic95_sup']:.2f}] adjetiva={est['adjetiva']}")


n=15 media=68.5 DT=13.977 SE=3.6089 t(gl=14)=2.1448
IC95=[60.76; 76.24] adjetiva=Regular


## Figuras (se renderizan al abrir, sin re-ejecutar)
![SUS por participante](../docs/mediciones/sus/figuras/fig-sus-por-participante.png)
![Respuestas por ítem](../docs/mediciones/sus/figuras/fig-sus-item-respuestas.png)
Generadas por `scripts/sus-analysis.py`. Detalle por participante en `P01.json`..`P15.json` y `ANALISIS-SUS.md`.
## Lectura
Media 68.5 (n=15, IC95 [60.76; 76.24]), apenas sobre la referencia 68 de Bangor: usabilidad *Regular*, consistente con cap1/cap2/cap8/cap9/cap12 del informe.